arch/arm/core/aarch64/mmu/arm_mmu.c

```diff
@@ -439,7 +440,12 @@ static int arm_mmu_init(const struct device *arg)
 	__asm__ volatile("mrs %0, sctlr_el1" : "=r" (val));
 	__ASSERT((val & SCTLR_M) == 0, "MMU is already enabled\n");
 
-	setup_page_tables(&kernel_ptables);
+	/*
+	 * Only core0 setup up the page tables.
+	 * TODO: allow Zephyr starts from other cores.
+	 */
+	if (!MPIDR_TO_CORE(GET_MPIDR()))
```

Comment:
Can you add a macro with a meaningful name instead of this? Something like `IS_BOOTING_CORE()` or something like that.